docs/app/interfaces/UsageProvider.ts

```diff
@@ -0,0 +1,9 @@
+import { Usage } from './Usage';
+
+export interface UsageProvider {
+    usage: Usage;
+}
+
+export function isUsageProvider(obj: any) {
```

Comment:
We can get rid of this usage provider. Instead just get the information from the json files we already use (eg. charts-page.json) this way we don't have to implement the interface every time etc..